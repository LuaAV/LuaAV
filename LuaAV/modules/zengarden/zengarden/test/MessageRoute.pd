#N canvas 183 229 1083 455 10;
#X obj 70 174 print x1;
#X obj 148 174 print x2;
#X obj 226 174 print x3;
#X obj 309 174 print x4;
#X obj 70 146 route 23 54 1;
#X msg 70 116 234 345 456;
#X msg 195 116 23 34 45;
#X msg 301 116 54 43;
#X msg 372 116 1 foo bar;
#X msg 465 116 walk the dog;
#X obj 492 18 loadbang;
#X obj 471 18 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X msg 641 114 impeach ringo starr;
#X obj 578 142 route big apple;
#X msg 822 115 apple pie;
#X msg 578 114 1 2 3;
#X msg 910 115 big apple pie;
#X obj 578 172 print y1;
#X obj 656 172 print y2;
#X obj 734 172 print y3;
#X obj 492 50 t b b b b b b b b b;
#X text 564 18 these examples are taken directly from the Pd documentation
;
#X connect 4 0 0 0;
#X connect 4 1 1 0;
#X connect 4 2 2 0;
#X connect 4 3 3 0;
#X connect 5 0 4 0;
#X connect 6 0 4 0;
#X connect 7 0 4 0;
#X connect 8 0 4 0;
#X connect 9 0 4 0;
#X connect 10 0 20 0;
#X connect 11 0 20 0;
#X connect 12 0 13 0;
#X connect 13 0 17 0;
#X connect 13 1 18 0;
#X connect 13 2 19 0;
#X connect 14 0 13 0;
#X connect 15 0 13 0;
#X connect 16 0 13 0;
#X connect 20 0 5 0;
#X connect 20 1 6 0;
#X connect 20 2 7 0;
#X connect 20 3 8 0;
#X connect 20 4 9 0;
#X connect 20 5 15 0;
#X connect 20 6 12 0;
#X connect 20 7 14 0;
#X connect 20 8 16 0;
