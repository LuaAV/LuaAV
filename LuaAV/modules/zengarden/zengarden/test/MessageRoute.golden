[@ 0.000ms] y1: apple pie
[@ 0.000ms] y2: pie
[@ 0.000ms] y3: impeach ringo starr
[@ 0.000ms] y3: 1 2 3
[@ 0.000ms] x4: walk the dog
[@ 0.000ms] x3: foo bar
[@ 0.000ms] x2: 43
[@ 0.000ms] x1: 34 45
[@ 0.000ms] x4: 234 345 456